recipe/meta.yaml

```diff
@@ -57,9 +57,13 @@ source:
     #########################################################################
     - ignore_libpython_macos.diff
 
+    # Added Sept 4 2020, but no new release since 2017
+    # https://github.com/ukoethe/vigra/commit/aaaf44d240aebd0eb3539b6e105a19a984000d81
+    - gcc9_compatibility.patch
+
 build:
   skip: true  # [win and py<35]
```

Comment:
I think we can drop the `skip`. Oldest supported Python version is 3.6. So the `skip` never occurs